src/main/java/io/matrix/bot/constants/accessor/MessageAccessor.java

```diff
@@ -0,0 +1,47 @@
+package io.matrix.bot.constants.accessor;
+
+import io.matrix.bot.constants.model.Error;
+
+import static io.matrix.bot.constants.Util.formatStackTraceString;
+
+public class MessageAccessor {
+
+    public static String getMessage(final Object message) {
+        if (message == null) {
```

Comment:
This looks like a good candidate for switch instead of if/else.

Reply:
I hate switch syntax in Java < 12 (AFAIR) version